content/docs/0.16.x/bridge/oauth/index.md

```diff
@@ -0,0 +1,398 @@
+---
+title: OpenID Authentication
+description: Enable/Disable OpenID authentication and implementation details for the identity provider
+weight: 20
+keywords: [0.11.x-bridge]
+aliases:
+  - /docs/0.11.x/operate/user_management/openid_authentication/
+---
+
+## Enable/Disable Authentication
+
+The Keptn Bridge contains a switch to enable/disable OpenID Connect-based authentication. This switch also enables session cookies. 
+
+You can enable/disable OpenID via the Helm Chart values when installing Keptn setting `enabled` to `true`:
```

Comment:
```suggestion
You can enable/disable OpenID via the Helm Chart values by setting `enabled` to `true` when installing Keptn:
```

Reply:
I just noticed that the configuration snippet that follows actually has `enabled` set to false -- isn't that misleading?